### PR TITLE
EWS: Improve speed of Facility Page Report

### PR DESCRIPTION
**Pull request by agrzywinski**
http://manage.dimagi.com/default.asp?152982#854271

### CODE REVIEW

File: corehq/pillows/mappings/user_mapping.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-USER_INDEX="hqusers_31n2pdyzqix4hd023lv6h67771c7wer"
+USER_INDEX="hqusers_31n2pdyzqix4hd023lv6h67771c7wes"
```
Comment (dimagimon) [inline corehq/pillows/mappings/user_mapping.py]: E225 missing whitespace around operator

Comment (czue) [inline corehq/pillows/mappings/user_mapping.py]: can you generate a new guid? visually it's really hard to tell these are different

File: custom/ewsghana/models.py [hunk 2]
```diff
@@ -57,3 +60,32 @@ def update_toggle(self):
...
             STOCK_AND_RECEIPT_SMS_HANDLER.set(self.domain, True, NAMESPACE_DOMAIN)
+
+
+class Numerator(fluff.Calculator):
+    @fluff.null_emitter
+    def numerator(self, doc):
+        yield None
+
+
+class EwsSmsUserFluff(fluff.IndicatorDocument):
```
Comment (czue) [inline custom/ewsghana/models.py]: can we use the ES index for this too?

File: corehq/pillows/mappings/user_mapping.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-USER_INDEX="hqusers_31n2pdyzqix4hd023lv6h67771c7wer"
+USER_INDEX="hqusers_14470f1ddab24c178dfcb3e0712055e"
```
Comment (dimagimon) [inline corehq/pillows/mappings/user_mapping.py]: E225 missing whitespace around operator

**Comment by agrzywinski** [comment]
@czue I also have done changes for this: http://manage.dimagi.com/default.asp?155103

**Comment by czue** [comment]
@agrzywinski sorry, this is now conflicting because i merged https://github.com/dimagi/commcare-hq/pull/5378. would you mind undoing the index changes and trying to build this on top of @esoergel's PR? in the future i'll try to work on avoiding these types of conflicts.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by agrzywinski** [comment]
@czue I updated PR

**Pull request merged**